docs/assets/dev/js/editor/document/elements/readme.md

```diff
@@ -0,0 +1,313 @@
+    ```
+    result: section deleted and all children elements.
+
+## Duplicate _Command_ -- `$e.run('document/elements/duplicate')`
+*  **Name**: Duplicate.
+*  **Description**: Duplicate container.
+*  **Returns**: `{Container | Container[]}` *Created container(s)*.
+*  **Arguments**: 
+
+   | Property     | Type             | Description |
+   |---           |---               |---|
+   | _container_  | `{Container}`    | Target.
+   | _containers_ | `{Container[]}`  | Targets.
+    
+* **Examples**:
+    Duplicate a section, Assuming we have this section, and want to duplicate it.
+    ![widget-heading](./_images/widget-heading.png)
+    ```javascript
+    // Get section container.
+    const eSection = elementor.getContainer('886643f');
+    
+    $e.run( 'document/elements/duplicate', { 
+        container: eSection,
+    } );
+    ```
+    Result: 
+    ![widget-heading-duplicated](./_images/widget-heading-duplicated.png)
+
+## Empty _Command_ -- `$e.run('document/elements/empty')`
+*  **Name**: Empty.
+*  **Description**: Delete all elements from the document.
+*  **Returns**: `{void}`
+
+    | Property | Type        | Requirement       | Description |
+    |---       |---          |---                |---|
+    | _force_  | `{Boolean}` | **optional**      | default: `{false}`, if true will delete all elements without confirmation.
+
+
+## Import _Command_ -- `$e.run('document/elements/import')`
+*  **Name**: Import.
+*  **Description**: Import elements to the document.
+*  **Returns**: `{Container | Container[]}` *Imported container(s)*.
+*  **Arguments**:
+
+    | Property  | Type               | Requirement       | Description |
+    |---        |---                 |---                |---|
+    | _model_   | `{Backbone.Model}` | **required**      | Template model.
+    | _data_    | `{Object}`         | **required**      | Data.
+    | _options_ | `{Object}`         | **optional**      | 
+
+    **_data:_**
+   
+    | Property      | Type       | Default   | Description                            |
+    |---------------|------------|-----------|----------------------------------------|
+    | content       | `{Object}` |           | Template content. 
+    | page_settings | `{Object}` |           | Template page settings.
+    
+    **_options:_**
+   
+    | Property         | Type                              | Default   | Description                            |
+    |------------------|-----------------------------------|-----------|----------------------------------------|
+    | withPageSettings | `{Boolean}`                       | `{false}` | Should set page settings of `args.page_settings`.
+    | at               | `{Number}`                        | `{null}`  | Position ( Automatically increased ).
+    | clone            | `{Boolean}`                       | `{false}` | Generate unique id for the model.
+    | edit             | `{Boolean}`                       | `{false}` | Is turn edit panel for the new element.
```

Comment:
Same as above

Reply:
Separation make it more understand able.